src/context/analysisResultStateSlice.ts

```diff
@@ -126,7 +126,9 @@ function generateTableFromApiData(
     );
 
     const name: string =
-      featureBoundary?.properties?.[adminLevelName] || 'No Name';
+      adminLayer.adminLevelNames
```

Comment:
Let's create a util function so that the tooltip and table can reuse the same and stay in sync